package.json

```diff
@@ -101,9 +101,9 @@
     "@storybook/addons": "^5.2.4",
```

Comment:
Are all of the offline dependency changes a result of the testing package updates?

Reply:
It's possible since many of these package updates are major versions, including the `jest` core library.